src/audit-license-headers.js

```diff
@@ -30,21 +30,22 @@ var repoutil = require('./repoutil');
 // constants
 var COMMON_RAT_EXCLUDES = [
     // Binary Files
-    '*.wav',
-    '*.webloc',
+    '(.*).wav',
+    '(.*).webloc',
 
     // iOS Related
-    '*.xcworkspacedata',
-    '*.xccheckout',
-    '*.xcscheme',
-    '*.xcodeproj',
-    '*-Info.plist',
+    '(.*).xcworkspacedata',
+    '(.*).xccheckout',
+    '(.*).xcscheme',
+    '(.*).xcodeproj',
+    '(.*)-Info.plist',
     'Info.plist',
 
     // Other
-    '.*',
-    '*.json', // Excludes all JSON files because commenting is not supported.
-    '*.txt', // Excludes all text files because commenting is not supported.
+    // eslint-disable-next-line no-useless-escape
```

Comment:
what's eslint's problem here?

Reply:
ESLint complains about the escaping being unless but it is actually needed in the command line.

**Example Command Line**
```bash
java -jar apache-rat-0.13.jar -d ./cordova-electron -e  "\.(.*)"
```

Reply:
ESLint didn't complain when I tested it. The escape isn't useless either.

Reply:
It might not be complaining now but it was before. I wouldn't be add a disable line randomly.

Reply:
Sorry, my comment before might have been very blunt. It might very well be that the ESLint rule had a bug that has since been fixed. Who knows...